tests/scheduler_test.rs

```diff
@@ -0,0 +1,60 @@
+use std::boxed::Box;
+use std::cell::RefCell;
+use std::rc::Rc;
+use yew::scheduler::{scheduler, Runnable, Scheduler};
+
+#[allow(missing_debug_implementations)]
+struct ImmediateRunnable {
+    output: Rc<RefCell<Vec<i32>>>,
```

Comment:
How about making `output` a `thread_local` so it doesn't need to be passed around? You could follow the `fn scheduler()` pattern